### PR TITLE
adding tasks at completed filter test was added

### PR DESCRIPTION
**Pull request by anna-fedrunova**
@maria-dyka, please, review

### CODE REVIEW

File: src/test/java/org/example/TasksLifeCycleTest.java [hunk 1]
```diff
@@ -27,6 +27,9 @@ public void smokeToDoMVC () {
         toggle("2");
         assertItemsLeft(3);
 
+        refresh();
+        sleep(1000);
```
Comment (anna-fedrunova) [inline src/test/java/org/example/TasksLifeCycleTest.java]: Why do you have to use sleep() here?

File: src/test/java/org/example/TasksOperationsAtCompletedFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Condition.*;
+import static org.example.pages.TodoMVCPage.*;
+
+public class TasksOperationsAtCompletedFilterTest {
+    @Test
+    public void addTest() {
+        add("1");
+        tasks.findBy(visible).shouldBe(empty);
```
Comment (anna-fedrunova) [inline src/test/java/org/example/TasksOperationsAtCompletedFilterTest.java]: Why to use findBy()?

File: src/test/java/org/example/TasksLifeCycleTest.java (hunk 1 above)
Comment (maria-dyka) [inline src/test/java/org/example/TasksLifeCycleTest.java]: Could you remove sleeps in your tests?

Comment (anna-fedrunova) [reply src/test/java/org/example/TasksLifeCycleTest.java]: No, because of the app's architecture. When it's fixed, I'll do that.

**Pull request merged**